areas/xdp-tutorial.org

```diff
@@ -58,6 +60,8 @@ XDP driver hook.
 This will demonstrate howto HW-offload packet header parse information via
 metadata area.
 
+BTF considerations, how can we transfer BTF info from HW-offloaded XDP-prog?
+
```

Comment:
BTF are not working with offloaded programs right now as far as I know. There seems to be a simple fix in the JIT to do, that's on the TODO list but not here yet.

(Also, spurious blank line.)